.travis.yml

```diff
@@ -18,13 +17,14 @@ install:
   - 'vsce package'  
 script:
   - 'npm test --silent'
+  - 'npm run ui-test --silent'
 after_success:
   - if [[ $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "master" ]]; then
     sonar-scanner;
     fi
 cache:
   directories:
-    - "node_modules"
+    - "$HOME/.m2"
```

Comment:
why the "node_modules" cache has been removed?

Reply:
I thought `opencv4nodejs-prebuilt` package was causing problems because it contains C++ addons. This is why I removed it from cache. However I have performed tests again with cached `node_modules` . Tests passed, so I am going to revert this change.